app/components/Sidebar/components/SharedDocumentLink.tsx

```diff
@@ -55,12 +66,6 @@ function DocumentLink(
     }
   }, [showChildren]);
 
-  React.useEffect(() => {
```

Comment:
Still need this otherwise we lose the automatic opening behavior as you click around?

Reply:
Ping on this question

Reply:
Sorry, don't get this? What is "automatic opening"? Expansion of document node in sidebar?

Reply:
Yes, the expansion as a document becomes active